var/spack/repos/builtin/packages/bazel/package.py

```diff
@@ -3,93 +3,135 @@
-    version('0.4.5', sha256='2b737be42678900470ae9e48c975ac5b2296d9ae23c007bf118350dbe7c0552b')
-    version('0.4.4', sha256='d52a21dda271ae645711ce99c70cf44c5d3a809138e656bbff00998827548ebb')
-
-    depends_on('java@8:', type=('build', 'link', 'run'))
-    depends_on('zip')
-
-    patch('fix_env_handling.patch', when='@:0.4.5')
-    patch('fix_env_handling-0.9.0.patch', when='@0.9.0:0.12.0')
-    patch('fix_env_handling-0.13.0.patch', when='@0.13.0:0.13.999')
-    patch('fix_env_handling-0.17.2.patch', when='@0.14.0:')
-    patch('link.patch')
-    patch('cc_configure.patch', when='@:0.4.5')
-    patch('unix_cc_configure.patch', when='@0.9.0')
-    patch('unix_cc_configure-0.10.0.patch', when='@0.10.0:0.14.999')
-    patch('unix_cc_configure-0.17.2.patch', when='@0.15.0:')
+    version('0.9.0',  sha256='efb28fed4ffcfaee653e0657f6500fc4cbac61e32104f4208da385676e76312a')
+    version('0.8.1',  sha256='dfd0761e0b7e36c1d74c928ad986500c905be5ebcfbc29914d574af1db7218cf')
+    version('0.8.0',  sha256='aa840321d056abd3c6be10c4a1e98a64f9f73fff9aa89c468dae8c003974a078')
+    version('0.7.0',  sha256='a084a9c5d843e2343bf3f319154a48abe3d35d52feb0ad45dec427a1c4ffc416')
+    version('0.6.1',  sha256='dada1f60a512789747011184b2767d2b44136ef3b036d86947f1896d200d2ba7')
+    version('0.6.0',  sha256='a0e53728a9541ef87934831f3d05f2ccfdc3b8aeffe3e037be2b92b12400598e')
+    version('0.5.4',  sha256='2157b05309614d6af0e4bbc6065987aede590822634a0522161f3af5d647abc9')
+    version('0.5.3',  sha256='76b5c5880a0b15f5b91f7d626c5bc3b76ce7e5d21456963c117ab711bf1c5333')
+    version('0.5.2',  sha256='2418c619bdd44257a170b85b9d2ecb75def29e751b725e27186468ada2e009ea')
+    version('0.5.1',  sha256='85e6a18b111afeea2e475fe991db2a441ec3824211d659bee7b0012c36be9a40')
+    version('0.5.0',  sha256='ebba7330a8715e96a6d6dc0aa085125d529d0740d788f0544c6169d892e4f861')
+    version('0.4.5',  sha256='2b737be42678900470ae9e48c975ac5b2296d9ae23c007bf118350dbe7c0552b')
+    version('0.4.4',  sha256='d52a21dda271ae645711ce99c70cf44c5d3a809138e656bbff00998827548ebb')
+    version('0.4.3',  sha256='cbd2ab580181c17317cf18b2bf825bcded2d97cab01cd5b5fe4f4d520b64f90f')
+    version('0.4.2',  sha256='8e6f41252abadcdb2cc7a07f910ec4b45fb12c46f0a578672c6a186c7efcdb36')
+    version('0.4.1',  sha256='008c648d3c46ece063ae8b5008480d8ae6d359d35967356685d1c09da07e1064')
+    version('0.4.0',  sha256='6474714eee72ba2d4e271ed00ce8c05d67a9d15327bc03962b821b2af2c5ca36')
+    version('0.3.2',  sha256='ca5caf7b2b48c7639f45d815b32e76d69650f3199eb8caa541d402722e3f6c10')
+    version('0.3.1',  sha256='218d0e28b4d1ee34585f2ac6b18d169c81404d93958815e73e60cc0368efcbb7')
+    version('0.3.0',  sha256='357fd8bdf86034b93902616f0844bd52e9304cccca22971ab7007588bf9d5fb3')
+
+    # https://docs.bazel.build/versions/master/install-compile-source.html#bootstrap-bazel
+    depends_on('java@8', type=('build', 'link', 'run'))
+    depends_on('python', type=('build', 'link', 'run'))
```

Comment:
Docs for 0.17-1.2 say both Python 2 and 3 work, no docs prior to 0.17... (can we not support ancient versions of bazel and TF?)

Reply:
That is totally fine for me!